front/src/lib/components/specialized/services/fields-duration.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+  import FieldSet from "$lib/components/display/fieldset.svelte";
+  import Notice from "$lib/components/display/notice.svelte";
+  import BasicInputField from "$lib/components/forms/fields/basic-input-field.svelte";
+  import type { Model, Service, ServicesOptions } from "$lib/types";
+  import { getModelInputProps } from "$lib/utils/forms";
+  import FieldModel from "$lib/components/specialized/services/field-model.svelte";
+
+  export let model: Model | undefined = undefined;
+  export let servicesOptions: ServicesOptions, service: Service;
+
+  function handleUseModelValue(fieldName: string) {
+    service[fieldName] = model ? model[fieldName] : undefined;
+  }
+
+  $: showModel = !!service.model;
+  $: fieldModelProps = model
+    ? getModelInputProps({
+        service,
+        servicesOptions,
+        showModel,
+        onUseModelValue: handleUseModelValue,
+        model,
+      })
+    : {};
+
+  $: totalHours =
+    isNaN(service.durationWeeklyHours) || isNaN(service.durationWeeks)
+      ? 0
+      : (service.durationWeeklyHours || 0) * (service.durationWeeks || 0);
+</script>
+
+<FieldSet title="Durée de la prestation">
+  <Notice titleLevel="h3" type="info">
+    <div>
+      Ceci correspond à la durée pendant laquelle les bénéficiaires vont être
+      mobilisés sur le service.
+    </div>
+    <div>
+      <strong>Exemples&nbsp;:</strong> Pour un atelier ponctuel de 3 h : temps hebdomadaire
+      = 3 heures, durée = 1 semaine (même si la durée réelle est inférieure, elle
+      sera arrondie à une semaine pour le calcul). Pour un accompagnement total de
+      30 h réparties sur 3 semaines : temps hebdomadaire moyen = 10 h, durée = 3
+      semaines.
+    </div>
+  </Notice>
+
+  <FieldModel {...fieldModelProps.durationWeeklyHours ?? {}}>
+    <BasicInputField
+      type="number"
+      id="durationWeeklyHours"
+      description="En nombre d'heures"
+      bind:value={service.durationWeeklyHours}
+    />
+  </FieldModel>
+
+  <FieldModel {...fieldModelProps.durationWeeks ?? {}}>
+    <BasicInputField
+      type="number"
+      id="durationWeeks"
+      description="En nombre de semaines. Si moins d’une semaine, laisser 1."
+      bind:value={service.durationWeeks}
+    />
+  </FieldModel>
+
+  <div class="hint-total-duration">
```

Comment:
Autant utiliser Tailwind CSS :)

```suggestion
  <div class="font-bold">
```

Et tu peux supprimer le CSS.

Reply:
Bien vu. Au départ il y avait plus de choses, mais à la fin il ne restait plus que le gras.